src/unistd.rs

```diff
@@ -2466,12 +2480,12 @@ impl User {
               libc::size_t,
               *mut *mut libc::passwd) -> libc::c_int
     {
-        let bufsize = match sysconf(SysconfVar::GETPW_R_SIZE_MAX) {
+        let cbuf_max = match sysconf(SysconfVar::GETPW_R_SIZE_MAX) {
             Ok(Some(n)) => n as usize,
             Ok(None) | Err(_) => 1024 as usize,
```

Comment:
Not that it's defined in any publicly visible place, but FreeBSD internally sets this limit to 1MB.